tests/actions.go

```diff
@@ -861,10 +861,10 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterConfig) error {
 		for _, pv := range pvList.Items {
 			afterPVNames = append(afterPVNames, pv.GetName())
 		}
-		klog.V(4).Info(afterPVNames)
+		k8se2elog.Logf("%v", afterPVNames)
 
 		if !reflect.DeepEqual(beforePVNames, afterPVNames) {
-			klog.Errorf("pv changed when we delete cluster: %s/%s, before: %v, after: %v",
+			k8se2elog.Logf("pv changed when we delete cluster: %s/%s, before: %v, after: %v",
```

Comment:
Can we add `ERROR` at the beginning of the log so that we can find the error logs easily?

Reply:
done